src/app/technical-record/body/body.component.spec.ts

```diff
@@ -1,26 +1,28 @@
-import { async, ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
-import { StoreModule } from '@ngrx/store';
-import { TechRecordHelpersService } from '@app/technical-record/tech-record-helpers.service';
-import { appReducers } from '@app/store/reducers/app.reducers';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { MatDialogModule } from '@angular/material/dialog';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MaterialModule } from '@app/material.module';
-import { SharedModule } from '@app/shared/shared.module';
-import { RouterTestingModule } from '@angular/router/testing';
-import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
-import { NgrxFormsModule } from 'ngrx-forms';
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { VehicleSummaryComponent } from '@app/technical-record/vehicle-summary/vehicle-summary.component';
-import { TechnicalRecordComponent } from '@app/technical-record/technical-record.component';
+import {async, ComponentFixture, getTestBed, TestBed} from '@angular/core/testing';
+import { StoreModule} from '@ngrx/store';
+import {TechRecordHelpersService} from '@app/technical-record/tech-record-helpers.service';
+import {appReducers} from '@app/store/reducers/app.reducers';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {MatDialogModule} from '@angular/material/dialog';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {MaterialModule} from '@app/material.module';
+import {SharedModule} from '@app/shared/shared.module';
+import {RouterTestingModule} from '@angular/router/testing';
+import {FontAwesomeModule} from '@fortawesome/angular-fontawesome';
+import {NgrxFormsModule} from 'ngrx-forms';
+import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
+import {TechnicalRecordComponent} from '@app/technical-record/technical-record.component';
```

Comment:
I think we should remove this TechnicalRecordComponent import  and also any non needed import